lib/api/addresses.js

```diff
@@ -532,6 +532,7 @@ module.exports = (db, server, userHandler, settingsHandler) => {
             // insert alias address to email address registry
             try {
                 r = await db.users.collection('addresses').insertOne(addressData);
+                db.database.collection('domaincache').insertOne({ domain: addressData.address.split('@')[1] });
```

Comment:
No `await` and no `catch()` handler for the Promise? This would be an uncaught exception if it fails